.travis.yml

```diff
@@ -4,6 +4,8 @@ php:
   - 5.5
   - 5.6
 
+sudo: true
```

Comment:
Please investigate a way to avoid using sudo. Using sudo results in the Travis tests being executed on a different/slower platform.